util/calc.py

```diff
@@ -41,26 +41,26 @@ def find_stream(stream_key, streams, distinct_sensors):
 
 
 @log_timing
-def get_particles(streams, start, stop, coefficients):
+def get_particles(streams, start, stop, coefficients,limit = None):
```

Comment:
One space after the comma, no spaces around the equals; see PEP 8 for details (https://www.python.org/dev/peps/pep-0008/)